midi_tokenizers/no_loss_tokenizer.py

```diff
@@ -41,7 +41,7 @@ def _build_vocab(self):
             self.vocab.append(f"VELOCITY_{vel}")
 
         time_vocab = self._time_vocab()
-        self.max_time_token = time_vocab[-1]  # Maximum time token
+        self.max_time_token: float = eval(time_vocab[-1][:-1])  # Maximum time token
```

Comment:
`eval` should be avoided if possible, maybe you should use `float`? Also, if it's a float, it's not a token 🤔